etc/base.yaml

```diff
@@ -564,3 +564,18 @@ velero:
         bucket: radar-base-backups
         config:
           s3Url: https://s3.amazon.com # protocol should be specified
+
+nifi:
+  _install: false
+  properties:
+    sensitiveKey: secret
+  auth:
+    singleUser:
+      username: username
+      password: password
```

Comment:
The secrets should be in base-secrets.yaml and if should be distinguished if it needs to be auto generated or should be manual.